packages/sdk-webos/src/deviceManager.ts

```diff
@@ -100,6 +97,56 @@ const parseDevices = (c: RnvContext, devicesResponse: string): Promise<Array<Web
     );
 };
 
+// Used for simulator
+const launchAppOnSimulator = async (c: RnvContext, appPath: string) => {
+    logTask('launchAppOnSimulator');
+
+    const webosSdkPath = getRealPath(c, c.buildConfig?.sdks?.WEBOS_SDK);
+
+    if (!webosSdkPath) {
+        return Promise.reject(`c.buildConfig.sdks.WEBOS_SDK undefined`);
+    }
+
+    const simulatorDirPath = path.join(webosSdkPath, 'Simulator');
+
+    const webOS_cli_version = await execCLI(c, CLI_WEBOS_ARES_LAUNCH, `-V`);
+
+    const regex = /\d+(\.\d+)?/g;
+    const webOS_cli_version_number = Number(webOS_cli_version.match(regex)?.at(0));
+    if (webOS_cli_version_number < 1.12) {
+        return logError(`Your webOS TV CLI version is ${webOS_cli_version_number}. You need to update it up to >=1.12`);
```

Comment:
isn't returning logError here allowing the process to go further instead of stopping right away? doublecheck it please and if that's the case `logError` has a second param, `isEnd`, if you set that to true it will stop everything